serverless/event_sources/serverless-listing-event-sources.adoc

```diff
@@ -10,8 +10,13 @@ You can use the `kn` CLI or the *Developer* perspective in the {product-title} w
 
 Currently, {ServerlessProductName} supports the following event source types:
 
-ApiServerSource:: Connects a sink to the Kubernetes API server.
-PingSource:: Periodically sends ping events with a constant payload. It can be used as a timer.
+API server source:: Connects a sink to the Kubernetes API server.
+Ping source:: Periodically sends ping events with a constant payload. It can be used as a timer.
+Sink binding:: Allows you to connect core Kubernetes resource objects, such as `Deployment`, `Job`, or `StatefulSet` objects, with a sink.
+Kafka source:: Connect a Kafka cluster to a sink as an event source.
```

Comment:
Same as the previous comnent. Don't we reference the exact Kubernetest objects/types? In this case it would be better to keep the original format: ApiServerSource and PingSource

Reply:
See above comment